src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -149,8 +147,6 @@
         </Style>
 
         <Style x:Key="LocationsListViewItemStyle" TargetType="ListViewItem">
-            <EventSetter Event="PreviewMouseLeftButtonDown" Handler="LocationsList_OnMouseLeftButtonDown"/>
```

Comment:
It was actually never needed; clicking or navigating in the list also sets the selected item, which automatically navigates to the location https://github.com/SonarSource/sonarlint-visualstudio/blob/cb70ace4f1784c42e45f9586bf0aa9d5b9c19b47/src/IssueViz/IssueVisualizationControl/ViewModels/IssueVisualizationViewModel.cs#L184